components/brave_extension/extension/brave_extension/background/api/cosmeticFilterAPI.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
can we squash this commit with the commit that removes the file? Otherwise we'll lose file history.

Reply:
done!